### DISCUSSION TITLE
There are no limits or efficiency issues to how much speed a harvester can carry

### DISCUSSION THREAD

**Discussion by wanming851** [Chia-Network/chia-blockchain#6702] [Support]
There are no limits or efficiency issues to how much speed a harvester can carry

**Comment by BrandtH22** [comment]
@wanming851 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
